packages/main/src/DynamicDateOption.ts

```diff
@@ -0,0 +1,20 @@
+import type { JsxTemplate } from "@ui5/webcomponents-base/dist/index.js";
+import type { DynamicDateRangeValue } from "./DynamicDateRange.js";
+
+/**
+ * @interface
+ * @public
+ * @since 2.0.0
+ */
+
+export interface IDynamicDateRangeOption {
```

Comment:
Maybe it would be cleanest to move this interface to the main DynamicDateRange.ts file and export it as a named export there